botfront/imports/api/importExport/export.methods.js

```diff
@@ -29,6 +29,17 @@ if (Meteor.isServer) {
     const { join, dirname, basename } = require('path');
     const fs = require('fs');
     
+    const nodegitTry = async (callback, ...args) => {
```

Comment:
I think what you could do is just export this part, and sprinkle it where needed. Exporting the whole try catch sounds inflexible, and maybe why you haven't actually used this function.

```js
if (process.env.MODE === 'development') {
    console.info(e)
}
```